src/js/enactors.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery.enactor");
+
+    /*
+     * Langauge enactor
+     */
+    fluid.defaults("gpii.firstDiscovery.enactor.lang", {
+        gradeNames: ["fluid.prefs.enactor", "autoInit"],
+        preferenceMap: {
+            "locale": {
```

Comment:
While making this change, sorry that I noticed I gave the wrong information. The messageLoader is actaully reading the locale value from a member option instead of the loader model so the model relay does not help here. Instead, I made the change to have messageLoader reads from a different setting path derived from the new namespaced language defintion: https://github.com/cindyli/first-discovery/blob/FLOE-314/src/js/firstDiscoveryEditor.js#L91-L95